bavard.go

```diff
@@ -22,6 +22,10 @@ import (
 	"rsc.io/tmplfunc"
 )
 
+const (
+	EnvFilter = "BAVARD" // environment variable to filter generation
```

Comment:
Maybe we can use a more descriptive variable key a la "BAVARD_GENERATE_ONLY" etc?